active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,572 @@
+# Teaching tip
+
+## Description
```

Comment:
(Minor) The first paragraph becomes the IntelliSense text, and this looks a bit long for IntelliSense text.

Reply:
See suggested code change on line below.